go.mod

```diff
@@ -76,13 +76,13 @@ require (
 	github.com/mdlayher/netlink v1.7.1
 	github.com/mdlayher/netx v0.0.0-20220422152302-c711c2f8512f
 	github.com/opencontainers/image-spec v1.1.0-rc2
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/runtime-spec v1.1.0-rc.1.0.20230215090456-58ec43f9fc39
```

Comment:
bumping this last time broke the build, is it fixed now?

Reply:
I think we should keep containerd's version

Reply:
Reverted.